suresh/loops.js

```diff
@@ -0,0 +1,317 @@
+//Loops for / while
+
+var arr = [1,2,3,4,5];
+
+//write a function that returns all odd numbers - [1,3,5]
+function oddNos (array) {
+  var oddNums = [];
+  var arrLength = array.length;
```

Comment:
Not necessary to set the length to a variable if it wont be used again.

Reply:
sure

Reply:
updated